Moggles/ClientApp/src/featureToggle/EditFeatureToggle.vue

```diff
@@ -152,7 +152,7 @@
 		},
 		data() {
 			return {
-				rowToEdit: null,
+				rowToEdit: '',
```

Comment:
I don't like this fix. The better fix would be add a null check before the line that is trying to read the isPermanent value on the rowToEdit object.

Reply:
Changes made...